4417_Sprint/04 - Xander/OCP CH3/MapDemo.java

```diff
@@ -24,12 +24,12 @@ public void populateproductsMap(HashMap<String,Integer> productsMap) {
     }
 
     public<K, V> void printKeyValuePairInMap(HashMap<K,V> map) {
-        System.out.println("\n"+"key & values");
+        System.out.println("\n" + "key & values");
         map.forEach((k,v) -> System.out.println("Key:" + k + " "+ "Value:" + v));
     }
 
     private <K, V, U> void printValueFromMap(HashMap<K,V> map, U key){
```

Comment:
U and K here are related right? So U either redundant or you should specify it to be a subtype of V?